docs/modules/ROOT/pages/access-control.adoc

```diff
@@ -204,3 +136,153 @@ TIP: A recommended configuration is to grant both roles to a secure governance c
+
+Although xref:api:access.adoc#AccessControl[`AccessControl`] offers a more dynamic solution for adding permissions to your contracts than Ownable, decentralized protocols tend to become more complex after integrating new contract instances and requires you to keep track of permissions separately in each contract. This increases the complexity of permissions management and monitoring across the system.
+
+image::access-control-multiple.svg[Access Control multiple]
+
+Protocols managing permissions in production systems often require more integrated alternatives to fragmented permissions through multiple `AccessControl` instances.
+
+image::access-manager.svg[AccessManager]
+
+The AccessManager is designed around the concept of role and target functions:
+
+* Roles are granted to accounts (addresses) following a many-to-many approach for flexibility. This means that each user can have one or multiple roles and multiple users can have the same role.
+* Access to a restricted target function is restricted to one role. A target function is defined by one https://docs.soliditylang.org/en/v0.8.20/abi-spec.html#function-selector[function selector] on one contract (called target). 
+
+For a call to be authorized, the caller must bear the role that is assigned to the current target function (contract address + function selector).
+
+image::access-manager-functions.svg[AccessManager functions]
+
+=== Using `AccessManager`
+
+OpenZeppelin Contracts provides xref:api:access.adoc#AccessManager[`AccessManager`] for managing roles across any number of contracts. The `AccessManager` itself is a contract that can be deployed and used out of the box. It sets an initial admin in the constructor who will be allowed to perform management operations.
+
+In order to restrict access to some functions of your contract, you should inherit from the xref:api:access.adoc#AccessManaged[`AccessManaged`] contract provided along with the manager so that the `restricted` modifier is available to protect any externally facing function. Note you will have to set the address of the AccessManager instance (xref:api:access.adoc#AccessManaged-constructor-address-[`initialAuthority`]) in the constructor so the `restricted` modifier can check permissions by calling the correct manager.
+
+Here's a simple example of an xref:tokens.adoc#ERC20[`ERC20` token] that defines a `mint` function that is restricted by an xref:api:access.adoc#AccessManager[`AccessManager`]:
+
+```solidity
+include::api:example$access-control/AccessManagedERC20MintBase.sol[]
+```
+
+NOTE: Make sure you fully understand how xref:api:access.adoc#AccessManager[`AccessManager`] works before using it or copy-pasting the examples from this guide.
```

Comment:
On the one hand I understand why we say that, but I also want devs to use this approach even if they don't fully understand how timers are handled (that is quite tricky).

Reply:
Yep, but I don't think there's an easy way, especially when there are competing delays (eg. execution delay vs admin operations delay).

The only idea I have is to enable tooling to compare how delays play out (eg. visualizing delays between role members, or visualizing the key dates after scheduling operations or granting roles)